wormchain/build/config/client.toml

```diff
@@ -1,5 +1,16 @@
-broadcast-mode = "block"
+# Tendermint client config
+
+###############################################################################
+###                           Client Configuration                            ###
+###############################################################################
+
+# The network chain ID
 chain-id = "wormchain"
+# The keyring's backend, where the keys are stored (os|file|kwallet|pass|test|memory)
 keyring-backend = "test"
-node = "tcp://localhost:26657"
+# CLI output format (text|json)
 output = "text"
+# <host>:<port> to Tendermint RPC interface for this chain
+node = "tcp://0.0.0.0:26657"
```

Comment:
this is a listen address but there should be a target ip here (0.0.0.0 != 127.0.0.1)

Reply:
`0.0.0.0` is the cosmwasm default. the `/build/config/` is for local development, where the devnet config is in `/devnet/`